libimage/image.go

```diff
@@ -669,32 +684,33 @@ func (i *Image) NamedRepoTags() ([]reference.Named, error) {
 	return repoTags, nil
 }
 
-// inRepoTags looks for the specified name/tag pair in the image's repo tags.
-func (i *Image) inRepoTags(namedTagged reference.NamedTagged) (reference.Named, error) {
+// inRepoTags looks for the specified name/tag in the image's repo tags.  If
+// `ignoreTag` is set, only the repo must match and the tag is ignored.
+func (i *Image) inRepoTags(namedTagged reference.NamedTagged, ignoreTag bool) (reference.Named, error) {
 	repoTags, err := i.NamedRepoTags()
```

Comment:
(Unrelated: the `NamedRepoTags` name is a bit misleading, it can return repo@digest values if that’s how an image was pulled.)

Reply:
Added a FIXME.   I think we inherited this name from Docker where RepoTags is used in some struct.